src/riscv-integration.adoc

```diff
@@ -122,7 +122,7 @@ cleared, if the encoding cannot represent the bounds exactly
 * <<CSETBOUNDSINEXACT>>: set the base and length of a capability. The base will be
 rounded down and/or the length will be rounded up if the encoding cannot represent
 the bounds exactly
-* <<CSEAL>>: seal capability
+* <<CSEALENTRY>>: seal entry capability
```

Comment:
IMHO, its best to describe this instruction as something like "seal capability as an entry" or something like that?